CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.3.7 (Next)
 * Your contribution here.
+* [#67](https://github.com/ashkan18/graphlient/pull/67): Lock activesupport version to < 6 - [@neroleung](https://github.com/neroleung).
```

Comment:
My 0.02c is that we don't want this in the changelog since it's locking ActiveSupport for tests only. This may be misleading to users who think that now somehow this gem cannot work with newer versions.

Reply:
@dblock, it's removed.